### PR TITLE
MTA-141 Renaming web console components 

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/MTA-141
MTA-6.1.0
Preview:
https://deploy-preview-669--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-6-ui-interface-views
and throughout the MTA documentation

### CODE REVIEW

File: docs/topics/mta-web-seeding-instances.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ The following parameters define applications, individuals, teams, verticals or a
 * Stakeholder groups
 * Job functions
 * Business services
-* Tag types
+* Categories
```
Comment (midays) [inline docs/topics/mta-web-seeding-instances.adoc:16]: can you please replace with **Tag categories** ?

File: docs/topics/mta-web-seeding-instances.adoc [hunk 2]
```diff
@@ -26,7 +26,7 @@ Stakeholders:
 
 Tags:
 
-. Create tag types
+. Create categories
```
Comment (midays) [inline docs/topics/mta-web-seeding-instances.adoc:29]: same here, can you please replace with **Tag categories** ?

File: docs/topics/mta-web-seeding-instances.adoc [hunk 3]
```diff
@@ -98,15 +98,15 @@ You can create a new business service by following the procedure below.
 ** Owner
 . Click *Create*.
 
-== Creating new tag types
+== Creating new categories
 
-{ProductName} ({ProductShortName}) uses tags in multiple categories and provides a list of default values. You can create a new tag type by following the procedure below.
+{ProductName} ({ProductShortName}) uses tags in multiple categories and provides a list of default values. You can create a new category by following the procedure below.
```
Comment (midays) [inline docs/topics/mta-web-seeding-instances.adoc:103]: can you please replace with " ... You can create a new **tag category** by following the ... "

File: docs/topics/mta-web-seeding-instances.adoc [hunk 4]
```diff
@@ -98,15 +98,15 @@ You can create a new business service by following the procedure below.
 ** Owner
 . Click *Create*.
 
-== Creating new tag types
+== Creating new categories
```
Comment (midays) [inline docs/topics/mta-web-seeding-instances.adoc:101]: can you please replace with "  Creating **new tag categories** "

File: docs/topics/mta-web-seeding-instances.adoc [hunk 5]
```diff
@@ -98,15 +98,15 @@ You can create a new business service by following the procedure below.
...
-{ProductName} ({ProductShortName}) uses tags in multiple categories and provides a list of default values. You can create a new tag type by following the procedure below.
+{ProductName} ({ProductShortName}) uses tags in multiple categories and provides a list of default values. You can create a new category by following the procedure below.
 
 .Procedure
 
-. In *Developer* view, click *Controls*.
+. In *Migration* view, click *Controls*.
 . Click *Tags*.
-. Click *Create tag type*.
+. Click *Create category*.
```
Comment (midays) [inline docs/topics/mta-web-seeding-instances.adoc:109]: please replace with " Click Create **tag category** "

File: docs/topics/mta-web-seeding-instances.adoc [hunk 6]
```diff
@@ -119,10 +119,10 @@ You can create a new tag, which is not in the default list, by following the pro
...
 .Procedure
 
-. In *Developer* view, click *Controls*.
+. In *Migration* view, click *Controls*.
 . Click *Tags*.
 . Click *Create tag*.
 . Enter the following information:
 ** Name
-** Tag type
+** Category
```
Comment (midays) [inline docs/topics/mta-web-seeding-instances.adoc:127]: can you please replace with " **Tag category** "

File: docs/topics/mta-web-adding-applications.adoc [hunk 7]
```diff
@@ -10,12 +10,12 @@ You can add an application to the *Application Inventory* for assessment and ana
 
 [TIP]
 ====
-Before creating an application, it is helpful to set up business services, check tags and tag types, and create additions as needed.
+Before creating an application, it is helpful to set up business services, check tags and categories, and create additions as needed.
```
Comment (midays) [inline docs/topics/mta-web-adding-applications.adoc:13]: can you please replace with **Tag categories**

**Review by midays — COMMENTED** [review]
Thank you for the effort, just a nitpick for a few changes, I think it would be better with these changes

File: docs/topics/mta-web-seeding-instances.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-seeding-instances.adoc:16]: Done

File: docs/topics/mta-web-seeding-instances.adoc (hunk 2 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-seeding-instances.adoc:29]: Done

File: docs/topics/mta-web-seeding-instances.adoc (hunk 3 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-seeding-instances.adoc:103]: Done

File: docs/topics/mta-web-seeding-instances.adoc (hunk 6 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-seeding-instances.adoc:127]: Done

File: docs/topics/mta-web-adding-applications.adoc (hunk 7 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-adding-applications.adoc:13]: Done

File: docs/topics/mta-web-seeding-instances.adoc (hunk 5 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-seeding-instances.adoc:109]: Done

File: docs/web-console-guide/master.adoc [hunk 8]
```diff
@@ -30,23 +30,23 @@ include::topics/about-the-user-interface.adoc[leveloffset=+2]]
...
 include::topics/mta-6-installing-web-console-on-openshift.adoc[leveloffset=+1]
 
 [id="configuring-the-instance-environment"]
 == Configuring the instance environment
 
-You can configure the following in *Administrator* view:
+You can configure the following in *Administration* view:
 
 * Credentials
 * Repositories
```
[the remark this answers is not in the document]
Comment (sbeskin-redhat) [reply docs/web-console-guide/master.adoc:52]: Added

**Comment by anarnold97** [comment]
@rromannissen & @dymurray - copying you in on the MTA 6.1.0 PRs.
Please can I ask for your comments?
Thanks

**Comment by sshveta** [comment]
Section for custom migration target is missing . May be that is in another PR ?

**Review by midays — APPROVED** [review]
Thank you for the change, looks great!

**Review by dymurray — APPROVED** [review]
LGTM assuming as @sshveta pointed out the custom migration target bits are in a separate PR (https://github.com/windup/windup-documentation/pull/676)

**Comment by sbeskin-redhat** [comment]
@sshveta Yes, custom migration targets are in another PR - https://github.com/windup/windup-documentation/pull/676

**Comment by HagayVider1** [comment]
Changed naming in 2 files:
web-console-guide/master.adoc
mta-web-administrator-view.adoc
See links below. I changed suffix of both files to .txt.
[master.txt](https://github.com/windup/windup-documentation/files/11343846/master.txt)
[mta-web-administrator-view.txt](https://github.com/windup/windup-documentation/files/11343847/mta-web-administrator-view.txt)

**Comment by HagayVider1** [comment]
No comment. All the changes look good.

**Pull request merged**